app/containers/MessageBox/forceJpgExtension.test.ts

```diff
@@ -0,0 +1,107 @@
+import { Image } from 'react-native-image-crop-picker';
+
+import { forceJpgExtension } from './forceJpgExtension';
+
+const attachment: Image = {
+	exif: null,
+	filename: 'IMG_0040.PNG',
+	path: 'tmp/temp',
+	height: 534,
+	width: 223,
+	data: null,
+	modificationDate: '1643984790',
+	localIdentifier: 'device/L0/001',
+	size: 16623,
+	sourceURL: '',
+	mime: 'image/jpeg',
+	cropRect: null,
+	creationDate: '1641490665'
+};
+
+describe('forceJpgExtension for iOS', () => {
```

Comment:
Why do we need this platform separation?

Reply:
Because the return of the function depends on which device the user is using

Reply:
WDYT?